apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1.Tests/SimpleSubscriberTest.cs

```diff
@@ -358,6 +366,58 @@ public void Dispose()
                 });
             }
         }
+
+        [Theory, CombinatorialData]
+        public void OneClientManyMsgs([CombinatorialValues(1, 2, 3, 4, 5)] int rndSeed)
+        {
+            // Test sending a large quantity of messages
+            const int maxStreamingPull = 14;
+            const int maxMoveNext = 14;
+            const int maxMsgCount = 80;
+            var rnd = new Random(rndSeed);
+            var streamingPullCount = rnd.Next(maxStreamingPull / 2, maxStreamingPull);
+            var msgs = Enumerable.Range(0, streamingPullCount).Select(rpcId =>
```

Comment:
I don't understand this bit, and I suspect I probably should...

Reply:
It's constructing a long-ish sequence of batches of messages; where all batches have varying sizes.
I'll add a comment